apps/hash-graph/.justfile

```diff
@@ -33,11 +33,30 @@ deployment-down *arguments:
 
 # Generates the OpenAPI client for the Graph REST API
 generate-openapi-client:
-  @just deployment-up graph --wait
-  @just yarn workspace @local/hash-graph-client-generator generate
-  @just yarn workspace @local/hash-graph-client prettier --write .
-  @just yarn workspace @local/hash-graph-client fix:eslint
-  @just deployment-down
+  #!/usr/bin/env bash
+  set -euo pipefail
+
+  just run server --openapi-only &
+
+  # When the script exits, clean-up and kill the server by searching for running processes with commands containing the
+  # `--openapi-only` flag
+  trap 'kill $(pgrep -f -- '\''--openapi-only'\'')' EXIT
+
+  retries=10
+
+  while ! just run server --healthcheck 2> /dev/null; do
```

Comment:
Currently not used but maybe it's more future proof to pass the flag here as well
```suggestion
  while ! just run server --healthcheck --openapi-only 2> /dev/null; do
```